src/ui/server/controllers/insurance/account/create/confirm-email-resent/index.ts

```diff
@@ -27,18 +27,14 @@ export const get = async (req: Request, res: Response) => {
       return res.redirect(ROUTES.PROBLEM_WITH_SERVICE);
     }
 
-    const urlOrigin = req.headers.origin;
+    const accountResponse = await api.keystone.account.get(id);
 
-    const account = await api.keystone.account.sendEmailConfirmEmailAddress(urlOrigin, id);
+    let accountEmail;
 
-    if (!account.success) {
-      console.error("Error sending new email verification for account creation and rendering 'new link sent' page");
-
-      return res.redirect(ROUTES.PROBLEM_WITH_SERVICE);
+    if (accountResponse && accountResponse.email) {
```

Comment:
if there's no account response, should it not immediately go to problem with service?

Reply:
Great shout, i've tweaked the controller to account for this and if there is no account response or email in the returned response, redirect to problem with service